proto/ssn/triton/grpc_service.proto

```diff
@@ -0,0 +1,1611 @@
+// Copyright 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Why "importing" all of this?

Reply:
We need this protodefinitions to communicate with triton

Reply:
Right, doesn't buf have these definitions for the "remote" generation? I know @kuceriklukas did something like that for our external imports... Or maybe triton generates these in go for us?

Reply:
**these

Reply:
(https://pkg.go.dev/github.com/sunhailin-Leo/triton-service-go/nvidia_inferenceserver#pkg-types) triton removed; found package

Reply:
Awesome, even better!